### ISSUE TITLE
Don't remap database

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/nmrium#1641]
The complexity of this code seems to high:

https://github.com/cheminfo/nmrium/blob/92d1c938562ba4f5004b6bf41bf62e4216e48079/src/data/data1d/database.ts#L153-L184

And it does not seem justified. We are expected to be able to display directly, without any change, the result of the database.

I think the table:

![image](https://user-images.githubusercontent.com/1484241/180450038-4d248191-6035-4193-82b6-fd983813b418.png)

Should be a one level row table and the cells containing delta, js could be a list of divs with fixed height.

Sorting in the database panel could also be removed.

**Comment by hamed-musallam** [comment]
@lpatiny 

I map the database data for two reasons:
https://github.com/cheminfo/nmrium/blob/92d1c938562ba4f5004b6bf41bf62e4216e48079/src/data/data1d/database.ts#L169-L178
1- we have to reshape the object in order to use rowspan so the table rows should be the signals
2- to hold an array in ids object for each record which contains the ids of the ranges and its signals for the highlight.

**Comment by lpatiny** [comment]
I don't think we currently display any highlight for ranges or signals in database. Am I wrong ?

And the first point I guess we could still construct the table from the original data without remapping all the table but if this seems too complex I can have a look at it.

**Comment by lpatiny** [comment]
Or maybe this is using the highlight system ?

![image](https://user-images.githubusercontent.com/1484241/183302765-009b9421-6000-48b8-8407-7ecc19ee297f.png)

**Comment by lpatiny** [comment]
If we need an 'id' in range I have nothing against adding it in the type definition and also in the database. We will probably go towards a uuid v4 as a general rule.
